examples/tutorials/cnndm_summarization.py

```diff
@@ -149,21 +149,76 @@ def apply_prefix(task, x):
 #
 # We can define a sequence generator to produce an output sequence based on the input sequence provided. This calls on the
 # model's encoder and decoder, and iteratively expands the decoded sequences until the end-of-sequence token is generated
-# for all sequences in the batch. The `greedy_generator` method shown below uses a greedy search (i.e. expands the sequence
-# based on the most probable next word).
+# for all sequences in the batch. The `generate` method shown below uses a beam search to generate the sequences.
```

Comment:
Should we mention somewhere that we use a beam size of 3 to generate summaries in this tutorial. Also might be worthwhile mentioning that larger beam sizes can result in better generation at the cost of computational complexity and that `beam_size == 1` is equivalent to a greedy decoder.